CONTRIBUTING.md

```diff
@@ -33,7 +32,7 @@ First, fork and clone the repository. If you’re not sure how to do this, pleas
 Run:
 
 ```bash
-npm install && npm run site:build:install
+npm install
```

Comment:
This being mentioned here didn't make any sense so I removed it (there's a section below about the docs site)